src/tcas/src/components/TcasComputer.ts

```diff
@@ -467,7 +467,7 @@ export class TcasComputer implements TcasComponent {
             // information, we need to rely on the fallback method
             // this also leads to problems above 1750 ft (the threshold for ground detection), since the aircraft on ground are then shown again.
             // Currently just hide all above currently ground alt (of ppos) + 380, not ideal but works better than other solutions.
-            const groundAlt = this.planeAlt - this.radioAlt; // altitude of the terrain
+            const groundAlt = this.planeAlt - this.radioAlt.value; // altitude of the terrain
```

Comment:
Looks like this `updateTraffic` function runs even if the selected radioAlt state is failure warning... probably the TCAS should change state in line 294 if no radio altitude is available? https://github.com/flybywiresim/a32nx/blob/870c4a82217b312a139826edf49f61e8591aee0b/src/tcas/src/components/TcasComputer.ts#L294 I didn't find a ref for this case so far however...

Reply:
True, I also guess the TCAS is inop with RA failure.

Reply:
You didn't actually make this any different though... just made it actually use the RA altitude... that's at least still a good improvement. 👍

Reply:
Yeah, this one can be merged regardless as it fixes a bug with functioning RAs (SSM=NO). TCAS INOP should already be properly working including ECAM on dual RA fault.